VariableLength[4] a